.github/pull_request_template.md

```diff
@@ -8,4 +8,5 @@ Add links to github/jira issues, design documents and other relevant resources (
 # Checklist
 - [ ] I have read the [contributing guidelines](../CONTRIBUTING.md).
 - [ ] I wrote/updated tests for new/changed code
+- [ ] I removed all sensitive data before every commit, including API endpoints or keys
```

Comment:
I think more natural English is "including API endpoints and keys" 😉

Reply:
fixed 🇬🇧🇺🇸